spaces.go

```diff
@@ -156,6 +168,24 @@ func (s *Space) Roles() ([]SpaceRole, error) {
 	return roles, nil
 }
 
+func (c *Client) CreateSpace(req SpaceRequest) (Space, error) {
+	buf := bytes.NewBuffer(nil)
+	err := json.NewEncoder(buf).Encode(req)
+	if err != nil {
+		return Space{}, err
+	}
+	r := c.NewRequestWithBody("POST", "/v2/spaces", buf)
+	resp, err := c.DoRequest(r)
+	if err != nil {
+		return Space{}, err
+	}
+	if resp.StatusCode != http.StatusCreated {
+		return Space{}, fmt.Errorf("CF API returned with status code %d", resp.StatusCode)
+	}
+	return c.handleSpaceResp(resp)
+
+}
+
```

Comment:
Thanks! I put it at this location because I wanted to place it nearer to the other `Spaces` methods but I might have accidentally left a gap.

Is there a convention on function placement on this project?